docker-compose.yml

```diff
@@ -22,26 +32,213 @@ services:
     ports:
       - "8081:8081"
+    environment:
+      - CLOUD_CONNECTOR_SERVICE_TO_SERVICE_CREDENTIALS={"config-manager":"swordfish","playbook-dispatcher":"swordfish"}
+      - CLOUD_CONNECTOR_MQTT_CLIENT_ID=api-service
+      - CLOUD_CONNECTOR_SLEEP_TIME_HACK=0
+      - CLOUD_CONNECTOR_MQTT_BROKER_TLS_SKIP_VERIFY=true
+      - CLOUD_CONNECTOR_INVENTORY_KAFKA_TOPIC=platform.inventory.host-ingress
+      - CLOUD_CONNECTOR_INVENTORY_KAFKA_BROKERS=kafka:9092
+      - CLOUD_CONNECTOR_CONNECTION_DATABASE_HOST=cloud-connector-db
+      - CLOUD_CONNECTOR_MQTT_BROKER_ADDRESS=tcp://mqtt-broker:1883
+    entrypoint: /cloud-connector api_server -l 0.0.0.0:8081
+  inventory-migrate:
+    image: quay.io/cloudservices/insights-inventory
+    depends_on:
+      - inventory-db
+    environment:
+      - INVENTORY_DB_HOST=inventory-db
+      - INVENTORY_DB_USER=insights
+      - INVENTORY_DB_PASS=insights
+      - INVENTORY_DB_NAME=inventory
+      - INVENTORY_LOG_LEVEL=DEBUG
+      - SQLALCHEMY_ENGINE_LOG_LEVEL=INFO
+    command: python manage.py db upgrade
+  inventory-api:
+    image: quay.io/cloudservices/insights-inventory
+    depends_on:
+      inventory-db:
+        condition: service_started
+      inventory-migrate:
+        condition: service_completed_successfully
+      kafka:
+        condition: service_healthy
+    environment:
+      - PAYLOAD_TRACKER_ENABLED=false
+      - INVENTORY_DB_HOST=inventory-db
+      - INVENTORY_DB_USER=insights
+      - INVENTORY_DB_PASS=insights
+      - INVENTORY_DB_NAME=inventory
+      - KAFKA_BOOTSTRAP_SERVERS=kafka:9092
+      - INVENTORY_LOG_LEVEL=DEBUG
+      - BYPASS_RBAC=true
+    ports:
+      - "8888:8888"
+    command: gunicorn -b :8888 run
+  inventory-mq:
+    image: quay.io/cloudservices/insights-inventory
     depends_on:
-      - kafka
-      - db
+      inventory-db:
+        condition: service_started
+      inventory-migrate:
+        condition: service_completed_successfully
+      kafka:
+        condition: service_healthy
+    environment:
+      - PAYLOAD_TRACKER_ENABLED=false
+      - INVENTORY_DB_HOST=inventory-db
+      - INVENTORY_DB_USER=insights
+      - INVENTORY_DB_PASS=insights
+      - INVENTORY_DB_NAME=inventory
+      - KAFKA_BOOTSTRAP_SERVERS=kafka:9092
+      - INVENTORY_LOG_LEVEL=DEBUG
+      - BYPASS_RBAC=true
+      - KAFKA_EVENT_TOPIC=platform.inventory.events
+    command: python inv_mq_service.py
+  playbook-dispatcher-db:
+    image: docker.io/postgres
+    restart: on-failure
     environment:
-      - CM_DB_HOST=db
-      - CM_DISPATCHER_IMPL=mock
-      - CM_CLOUD_CONNECTOR_IMPL=mock
-      - CM_INVENTORY_IMPL=mock
-      - CM_PLAYBOOK_HOST=localhost
+      POSTGRES_USER: insights
+      POSTGRES_PASSWORD: insights
+      POSTGRES_DB: insights
+  playbook-dispatcher-migrate:
+    image: quay.io/cloudservices/playbook-dispatcher
+    entrypoint: "/app migrate up"
+    depends_on:
+      - playbook-dispatcher-db
+    environment:
+      CLOWDER_ENABLED: false
```

Comment:
Wrap the value in quotes to prevent the yaml parser from interpreting this as a boolean:

```yaml
CLOWDER_ENABLED: "false"
```

[ref](https://docs.docker.com/compose/compose-file/#environment)

Reply:
Nice catch. I updated all the truthy values so they get passed as strings through to the environment.